webdriverio/web/examples/typescript/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "wdio run ./wdio.conf.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@wdio/cli": "^7.1.2"
```

Comment:
This should be a devDependency

Reply:
@wswebcreation that's how `wdio cli` installed it. You still want me to change it?

Reply:
Then you installed it with the `save` option like `npm install --S @wdio/cli` instead of `npm install --D @wdio/cli`. It's a dev dependency so yeah, please put it in the devDependencies and remove this one